### ISSUE TITLE
Flixtor will be back

### ISSUE THREAD

**Issue by gtuk** [TorrentLookup/Flixtor#86]
Dear Flixtor users,
Flixtor will be back. Not the official version created by torrentlookup but an unofficial fork. I completely change the api calls and rewrite some parts of the app. It's now working with the yify and eztv apis. You can get the first working version here https://github.com/gtuk/Flixtor-1/tree/relaunch
In the future i will create some new features that were missing in the official version of Flixtor. Bug reports are welcome

**Comment by gtuk** [comment]
@Draxler 
i develop an installer for the new flixtor app but at the moment its not finished sothat you must built it with the source code. 
Here are a quick overview how to built it

Install
git clone https://github.com/gtuk/Flixtor-1/tree/relaunch

Node / NPM
Install nodejs on your computer http://nodejs.org/

Dependencies
Run 'npm install' in command prompt on the folder you clone the project. This will create a node_modules folder containing all the dependencies you need to have if you want to run the application.

Node-webkit
Download the latest version of Node-Webkit https://github.com/rogerwang/node-webkit
Extract & copy files inside your project folder.
Replace 'ffmpegsumo' of node-webkit by one inside the librairies folder of the project.

Start
Start the flixtor app e.g on linux with
"< path to node webkit > < flixtor folder >"

I don't own the official repo of flixtor my page is only an fork. You can't post bugs the, sorry. If you have any bugs you can leave them here in the official repository. Time to time i will look here for bugs related to my version. 
An android version is also planned but this takes more time because i didn't have the code base from the original app sothat i must built it from my own but the base is finished, only design issues now

Hope you get my version working, otherwise let me know maybe i can i help you.

**Comment by tartane** [comment]
This is awesome! we will definitely post it on our facebook page as soon as it is done!

**Comment by gtuk** [comment]
@tartane 
maybe it is possible to use this official flixtor repo for the future?

**Comment by tartane** [comment]
@gtuk 
of course it is possible, we can work something out

**Comment by gtuk** [comment]
Sounds great. Is any one of the old developers interested to help? At the moment i have some problem with the grunt-node-webkit-builder on linux

**Comment by gtuk** [comment]
Yes I'm sure I can add on installer for all distros. Hope I get i
working on weekend then i have a little bit more time
 Sent using Hushmail
On 10.7.2014 at 11:39 PM, "Draxler"  wrote:

```
Any chance to get Flixtor build for Win ? 

—
```

**Comment by gtuk** [comment]
@Draxler 
thanks for this great information. It would be aweseome if you cat put these informationen more into public. That would be mutch easier for me because I didn't have the time to create multiple forums thread. But maybe you can still wait if we merge my fork here into the official repository if the installer is finished. Hope to get it working tomorror or on sunday.
Also thanks for you time and feedback

**Comment by tartane** [comment]
I can MAYBE get an article on torrentfreak once everything is up and running.

**Comment by gtuk** [comment]
thanks for the info. I always have an backup (local working copy) because i know these assholes ;)
First version (linux, mac, windows) is up in 10min. Can someone please test if everything works fine

**Comment by gtuk** [comment]
It's done!
win:
http://www.share-online.biz/dl/10EJ8K8N8SZ6

mac:
http://www.share-online.biz/dl/POGM8K8N105

linux64:
http://www.share-online.biz/dl/XCYP8K8N1D

linux32:
http://www.share-online.biz/dl/FA7T8K8NH46B

Install note: 
- Download
- Unzip
- Start executable

Info: This is only a first minimalistic version. In the next days i rewrite the subtititle functionality and fix some bugs. Hope to get feedback if something is wrong

**Comment by gtuk** [comment]
@Draxler you can download as a free user on share-online. Hmm or maybe its a problem of you locale I will add it on another hoster

**Comment by gtuk** [comment]
@Draxler i didn't know these restrictions i will add it on mega

**Comment by gtuk** [comment]
@tartane do you have the original source code of the flixtor android app? I developed an android from the base but i think if we can merge the two apps and use the old flixtor app design i would be nicer

**Comment by tartane** [comment]
@gtuk 
Yes, I have the original source code... I'll put it on github tonight

**Comment by gtuk** [comment]
thats awesome, thanks

**Comment by gtuk** [comment]
Did anyone try the uploaded flixtor releases?

**Comment by gtuk** [comment]
Is it working correct on your windows computer?

**Comment by gtuk** [comment]
excellent, thanks

**Comment by gtuk** [comment]
@tartane any possibility to contact you wihout github? Mail? Irc?

**Comment by gtuk** [comment]
@Draxler 
the last night i fixed/change some stuff in the app. Maybe tomorrow i will upload a new version for testing. You and @pepijn809 a welcome to spread out the news if everything is in order in the next weeks. I should not be the main player ;) it would be nicer for me if somebody can post news, because than i can focussed complete on development. 

I have also some backups from current state, sothat i think its no problem if github shut this repo also down

@pepijn809 
Yes anime is definitely planned for the future, but first thing is to get an save code base for further development. 
My dev system has an settings page in the current state but it's not finished yet. I don't like the current design. 

@Draxler @tartane @pepijn809 @Japoilski 
Can we create an irc channel somewhere sothat we have an discussion place?

**Comment by pepijn809** [comment]
:)

**Comment by gtuk** [comment]
@pepijn809  
Can you create an irc for us if you are back from work?
That would be great

**Comment by pepijn809** [comment]
Good to see, @Draxler nice effort man!

The Creation of PopcornTime and Flixtor is amazing. It shows that we (people of the world) do care about service, quality and the fact that we notice that companies no longer seem to really focus on the consumer. But rather on making more money, which is happening in many different business branches all over the world. 

We can't solve it in a day, just as Rome wasen't build in a day. But, we can however work on alternatives, which may be seen as "Grey" when coming to legal areas. But the goal isn't to replace netflix, hulu or anything similar. It's to find something worth replacing, on both sides. 

I do care about the creators, whether they are independent or bigger companies. I care. 
You, as a creator have the power to let others in your branche know what the problems are, maybe they will listen, maybe not. 

But don't make the mistake to think that you can defeat the development of p2p related systems (and some piracy in general). It's true some people will never stop pirating, they just dont wanna pay. 

But for the most of us, it's a different reason. And where you may shoot down one Popcorn or Shut down a Flixtor. Another will come up, it's a true cat and mouse game. 

Maybe it's time for the bigger companies (because these are mostly affected, and are responsible for most of the new anti piracy laws) to at least look at what the  "less legal/illegal market" is doing, and asside from "being free", why it's so darn popular. 

I've seen some companies that profited from looking to piracy, and building with that knowledge. You can change rules, we can't from here. Even some PopcornTime devs said they would join if you where only offering a job, most of us are not aimed at the piracy aspect of the whole situation. 

We're rooted differently.

**Comment by gtuk** [comment]
Can we create an area in reddit like the popcorn time section or is something like that still exist?

**Comment by pepijn809** [comment]
Creating a "Platform" where a community and a AI set out the simple rules, may create a p2p marketplace where anyone can join, create ,share and make use of. Including independent creators, as much as giant companies. Where there is demand for a movie, there is people sharing it. 

If people share the content that they want, after maybe a initial seedbox system to start something off and let it die if it doesn't attract enough. That, my dear people. may be a key to the future of on demand content. 

This will benefit everyone, except the greedy people.
Prices will be much lower due the platform existing out of already implace technology, including technologies in the homes of people. 

There could potentially be a pricepoint for certain movies, no fixed pricepoint for big companies. Everyone is equel, and demand is what is dynamicly in charge of the price. 
The more demand... Well, these rules can be thought of, if anyone is taking the time to make it. 

I just feel like if the platform is easily implemented, maintened by no one, after initial creation ( a bit like creating bitcoins ) and after that is solely depending upon the community to be driven. Than just maybe, can we create a marketplace, a platform for the many. 

Do note that many in the daily monopoly of content will not like the idea. Basicly because they are already in a goldmine, but what if it's able to provide the best for all. And still give a decent income, even for the big companies, would there be more focus on that system? if, let's say it would be supported by all?

Well, you can see how much impact PopcornTime and Flixtor have. How many apps, websites, file lockers etc. host or somehow mirror/distrubute/make content available for mainstream. 

What would happen if we turn this amazing experience to the mainstream, with their help "and fair prices". 

We can never eliminate piracy, but we can surely minimize it.

**Comment by gtuk** [comment]
Can you see the owner of /r/flixtor? Maybe its tartane, then i will take with him

**Comment by pepijn809** [comment]
Censorship and blockades rarely prove to be effective.

**Comment by gtuk** [comment]
Hmm maybe its possible to contact him. Tartane is the owner of this github repo

**Comment by pepijn809** [comment]
we love netflix

**Comment by gtuk** [comment]
Yes thats true but the first thing we can do is ask him first if he want to change the owner if the repo we should not say that flixor will be back

**Comment by gtuk** [comment]
please come to irc i think its better than talk here

**Comment by pepijn809** [comment]
Open Source may be the key to the future

**Comment by andresgmh** [comment]
@tartane hello.

I speak spanish, I wil try speak a good english. Please it's possible get a copy of the original source of the flixtor android app? I would like develop an Android app for educational purposes. Please let me know how to get a copy! Thanks!!

**Comment by pepijn809** [comment]
i wish you all the best.

**Comment by andresgmh** [comment]
@pepin809 yes, I am interested in that new Flixtor Android app. How can you put it touch me with your dev team. I will wait that @tartane reply my msg too.

Thanks

**Comment by FlixtorMe** [comment]
Hi @andresgmh i think tartane will not reply he isn't part of the new flixtor team. We are working on an Android port. If you have interest to get in touch with us please leave a message info@flixtor.me
